The doc subpkg of tiff fails the doc test

The documentation subpackage of tiff fails testing with the following error:

```
2025/02/04 22:46:25 INFO where option is:
2025/02/04 22:46:25 INFO  -M size       set the memory allocation limit in MiB. 0 to disable limit.
2025/02/04 22:46:25 WARN tiffsplit: invalid option -- '-'
2025/02/04 22:46:25 WARN + tiffsplit --help
2025/02/04 22:46:25 INFO
2025/02/04 22:46:25 INFO
2025/02/04 22:46:25 INFO LIBTIFF, Version 4.7.0
2025/02/04 22:46:25 INFO Copyright (c) 1988-1996 Sam Leffler
2025/02/04 22:46:25 INFO Copyright (c) 1991-1996 Silicon Graphics, Inc.
2025/02/04 22:46:25 INFO
2025/02/04 22:46:25 INFO Split a multi-image TIFF into single-image TIFF files
2025/02/04 22:46:25 INFO
2025/02/04 22:46:25 INFO usage: tiffsplit [option] input.tif [prefix]
2025/02/04 22:46:25 INFO where option is:
2025/02/04 22:46:25 INFO  -M size       set the memory allocation limit in MiB. 0 to disable limit.
2025/02/04 22:46:25 WARN tiffsplit: invalid option -- '-'
2025/02/04 22:46:25 WARN + exit 0
2025/02/04 22:46:25 INFO running test pipeline for subpackage tiff-doc
2025/02/04 22:46:25 INFO building test workspace in: '/home/user/tmp/melange-guest-1677200726-tiff-doc' with apko
2025/02/04 22:46:25 [DEBUG] GET https://packages.wolfi.dev/os/apk-configuration
2025/02/04 22:46:25 INFO setting apk repositories: [/home/user/src/wolfi-os/packages https://packages.wolfi.dev/os]
2025/02/04 22:46:25 INFO image configuration:
2025/02/04 22:46:25 INFO   contents:
2025/02/04 22:46:25 INFO     build repositories: []
2025/02/04 22:46:25 INFO     runtime repositories: []
2025/02/04 22:46:25 INFO     keyring:      []
2025/02/04 22:46:25 INFO     packages:     [tiff-doc apk-tools grep man-db texinfo]
2025/02/04 22:46:25 INFO installing ca-certificates-bundle (20241121-r1)
2025/02/04 22:46:25 INFO installing wolfi-baselayout (20230201-r16)
2025/02/04 22:46:25 INFO installing glibc (2.40-r8)
2025/02/04 22:46:25 INFO installing ld-linux (2.40-r8)
2025/02/04 22:46:25 INFO installing libgcc (14.2.0-r8)
2025/02/04 22:46:25 INFO installing glibc-locale-posix (2.40-r8)
2025/02/04 22:46:25 INFO installing zlib (1.3.1-r5)
2025/02/04 22:46:25 INFO installing libcrypto3 (3.4.0-r6)
2025/02/04 22:46:25 INFO installing libssl3 (3.4.0-r6)
2025/02/04 22:46:25 INFO installing apk-tools (2.14.4-r2)
2025/02/04 22:46:25 INFO installing libpcre2-8-0 (10.44-r2)
2025/02/04 22:46:25 INFO installing grep (3.11-r2)
2025/02/04 22:46:25 INFO installing libpipeline (1.5.8-r0)
2025/02/04 22:46:25 INFO installing libbz2-1 (1.0.8-r10)
2025/02/04 22:46:25 INFO installing libxcrypt (4.4.38-r0)
2025/02/04 22:46:25 INFO installing libcrypt1 (2.40-r8)
2025/02/04 22:46:25 INFO installing perl (5.40.1-r0)
2025/02/04 22:46:25 INFO installing libstdc++ (14.2.0-r8)
2025/02/04 22:46:25 INFO installing groff (1.23.0-r3)
2025/02/04 22:46:25 INFO installing libseccomp (2.6.0-r0)
2025/02/04 22:46:25 INFO installing gdbm (1.24-r2)
2025/02/04 22:46:25 INFO installing man-db (2.13.0-r1)
2025/02/04 22:46:25 INFO installing ncurses-terminfo-base (6.5_p20241228-r0)
2025/02/04 22:46:25 INFO installing ncurses (6.5_p20241228-r0)
2025/02/04 22:46:25 INFO installing texinfo (7.2-r0)
2025/02/04 22:46:25 INFO installing tiff-doc (4.7.0-r1)
2025/02/04 22:46:25 INFO installing wolfi-keys (1-r8)
2025/02/04 22:46:25 INFO installing busybox (1.37.0-r0)
2025/02/04 22:46:25 INFO installing wolfi-base (1-r6)
2025/02/04 22:46:26 INFO OCI layer digest: sha256:6ba13e615b5be1889e5a6da03145226ecaa9ca8cdcdf252ed0d8207f0f2b7afb
2025/02/04 22:46:26 INFO OCI layer diffID: sha256:d925a398bbd7937611ae71ef88ef5f4cd2e81c1cd793d749a18ba2791dc249e2
2025/02/04 22:46:26 INFO saving OCI image locally: apko.local/cache:c7493564f733ee9926aa776282cdcff534a14707b15be54fb2e9abf7efee077f
2025/02/04 22:46:27 WARN skipping local domain tagging apko.local/cache:c7493564f733ee9926aa776282cdcff534a14707b15be54fb2e9abf7efee077f as index.docker.io/library/melange:latest
2025/02/04 22:46:27 INFO running step "test/docs"
2025/02/04 22:46:27 WARN + '[' -d /home/build ]
2025/02/04 22:46:27 WARN + cd /home/build
2025/02/04 22:46:27 WARN + exit 0
2025/02/04 22:46:27 INFO running step "docs readability check"
2025/02/04 22:46:27 INFO See:
2025/02/04 22:46:27 INFO tiff-doc-4.7.0-r1 contains:
2025/02/04 22:46:27 INFO var/lib/db/sbom/tiff-doc-4.7.0-r1.spdx.json
2025/02/04 22:46:27 INFO
2025/02/04 22:46:27 INFO This package [tiff-doc] is completely empty (i.e. installs no files).
2025/02/04 22:46:27 INFO Please check the package build for proper docs installation, and either:
2025/02/04 22:46:27 INFO   (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or
2025/02/04 22:46:27 INFO   (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or
2025/02/04 22:46:27 INFO   (c) remove the docs subpackage entirely
2025/02/04 22:46:27 WARN + '[' -d /home/build ]
2025/02/04 22:46:27 WARN + cd /home/build
2025/02/04 22:46:27 WARN + basename /home/build/melange-out/tiff-doc
2025/02/04 22:46:27 WARN + doc_pkg=tiff-doc
2025/02/04 22:46:27 WARN + apk info -L tiff-doc
2025/02/04 22:46:27 WARN + wc -l
2025/02/04 22:46:27 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:46:27 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:46:27 WARN + '[' 3 -le 3 ]
2025/02/04 22:46:27 WARN + echo See:
2025/02/04 22:46:27 WARN + apk info -L tiff-doc
2025/02/04 22:46:27 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:46:27 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:46:27 WARN + echo 'This package [tiff-doc] is completely empty (i.e. installs no files).'
2025/02/04 22:46:27 WARN + echo 'Please check the package build for proper docs installation, and either:'
2025/02/04 22:46:27 WARN + echo '  (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or'
2025/02/04 22:46:27 WARN + echo '  (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or'
2025/02/04 22:46:27 WARN + echo '  (c) remove the docs subpackage entirely'
2025/02/04 22:46:27 WARN + exit 1
2025/02/04 22:46:28 INFO pod c93df4668ce6d010d14879b3df51b516f52686dc782dc953a34319a2c1128ebb terminated
2025/02/04 22:46:28 INFO pod 496c576ed648d03ac6d1dcac16101912401ed1f2f95f0197001c434c5706232d terminated
2025/02/04 22:46:28 INFO ERROR: failed to test package. the test environment has been preserved:
2025/02/04 22:46:28 INFO   workspace dir: /home/user/tmp/melange-workspace-1519158437
2025/02/04 22:46:28 INFO   guest dir: /home/user/tmp/melange-guest-1677200726
2025/02/04 22:46:28 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:121: test/tiff] Error 1
```

--- a/tiff.yaml
+++ b/tiff.yaml
@@ -1,7 +1,7 @@
 package:
   name: tiff
   version: 4.7.0
-  epoch: 1
+  epoch: 2
   description: Provides support for the Tag Image File Format or TIFF
   copyright:
     - license: libtiff
@@ -45,7 +45,13 @@ subpackages:
   - name: tiff-doc
     pipeline:
       - uses: split/manpages
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/share
+          mv "${{targets.destdir}}"/usr/share/doc "${{targets.contextdir}}"/usr/share
     description: tiff manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
   - name: tiff-dev
     pipeline:

